### PR TITLE
Forces the temporary file deletion when user uses a security rm alias

### LINKED ISSUE #623: Using rm -i option  (prompt before every removal)
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] I looked for duplicate issues before submitting this one

### Description

As a use a security alias `alias rm="rm -i"`, each time I execute a linux command, oh_my_posh ask me to:  `rm: remove regular file '/tmp/ojullien.start.1874365`

Possible correction is to use the -f option `rm -f "$TIMER_START"` on the related source file [omp.bash](https://github.com/JanDeDobbeleer/oh-my-posh/tree/main/src/init/omp.bash) lines: 15 and 27

### Environment

- Oh my Posh version: 3.133.1
- Theme: 3.133.1
- Operating System: Ubuntu 20.10 Linux 5.8.0-49-generic x86-64
- Shell: GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
- Terminal: xfce terminal

### Steps to Reproduce

1. Follow the [official documentation](https://ohmyposh.dev/docs/installation) to install oh_my_posh on ubuntu
2. reload your profile for the changes to take effect. `. ~/.bashrc`
3. use ls (or another command)

**Expected behavior:** Execute the command.

**Actual behavior:** Execute the command then ask me to: `rm: remove regular file '/tmp/ojullien.start.1874365`

### PR DESCRIPTION
**Pull request by ojullien**
Signed-off-by: ojullien <3778194+ojullien@users.noreply.github.com>

### Prerequisites

- [X] I have read and understand the `CONTRIBUTING` guide
- [X] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

Added the -f option to the rm command to override the security alias `alias rm="rm -i"`

Fix #623 
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
@ojullien would gladly merge this when you can rename the commit to respect the conventional commit guidelines.

**Comment by JanDeDobbeleer** [comment]
You now have 3 commits rather than 1, I can merge this for you if you want :-) I'll need to offer some Git lessons for free (no offence, just something I think a lot of people would benefit from).

**Review by JanDeDobbeleer — APPROVED** [review]

**Comment by ojullien** [comment]
Sorry, the commits escaped me :).
I was waiting the end of macos check before the merge.
I've tested Oh my Posh version: 3.133.1 on my macos, everything is fine.
I don't know with my aliases are expanded on ubuntu and not in macos

**Pull request merged**
